scalafix/input/src/main/scala/fix/Collectionstrawman_v0_TupleNZipped.scala

```diff
@@ -0,0 +1,22 @@
+/*
+rule = "scala:fix.Collectionstrawman_v0"
+ */
+package fix
+
+object Collectionstrawman_v0_Tuple2Zipped {
+  def zipped(xs: List[Int], ys: List[Int]): Unit = {
+    (xs, ys).zipped
+    (coll(1), coll(2)).zipped
```

Comment:
I recommend you add at least one test with comments inside of the tuple, the biggest reason scalafix exposes low-level token APIs is so that trivia like comments and formatting can be preserved.

Reply:
done